Libplanet/Net/DifferentAppProtocolVersionException.cs

```diff
@@ -24,6 +27,7 @@ internal sealed class DifferentAppProtocolVersionException : Exception
         /// </param>
         public DifferentAppProtocolVersionException(
             string message,
+            byte[] identity,
```

Comment:
I guess `identity` isn't useful because it's a temporary value for `NetMQRouter`.

Reply:
I've added this parameter just for use in [here](https://github.com/planetarium/libplanet/pull/949/files#diff-9045fd925fccb8bca88db49a0ab95d0cR600-R608).

Reply:
🆗